src/Components/Home.js

```diff
@@ -5,10 +5,10 @@ import { NavigationContext } from '../Context/NavigationContext'
 import { useQuery, gql } from "@apollo/client"
 
 const GET_CRYPTIDS = gql`
-    {
+    query GetCryptids {
```

Comment:
I'll remember to add this for next time I use circle CI

Reply:
Me too

Reply:
^^^